products/laravel.md

```diff
@@ -19,9 +19,9 @@ releases:
 -   releaseCycle: "9"
     support: 2023-08-08
     eol: 2024-02-08
-    latest: '9.47.0'
```

Comment:
It's not clear that this page tracks https://github.com/laravel/framework and not https://github.com/laravel/laravel (versions are not the same).

And maybe the version that should be followed is https://github.com/laravel/laravel, because https://github.com/laravel/framework states :

> Note: This repository contains the core code of the Laravel framework. If you want to build an application using Laravel, visit the main [Laravel repository](https://github.com/laravel/laravel).

Reply:
#2318